### PR TITLE
added first crypto bank

### PR DESCRIPTION
**Pull request by alphadeveth**

### CODE REVIEW

**Comment by llamatester** [comment]
The raydium adapter exports: 
        
 
 _______________________________________
    Dexs for 23/12/2023
    _______________________________________
    
    SOLANA 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Daily volume: 667.15 M
    Timestamp: 1703375998
    
    
    
    
    Done in 3.39s.

**Comment by llamatester** [comment]
The first-crypto-bank adapter exports: 
        
 
 _______________________________________
    Fees for 23/12/2023
    _______________________________________
    
    Error: connect ECONNREFUSED ::1:5432
        at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
        at cachedError (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/postgres/cjs/src/query.js:171:23)
        at new Query (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/postgres/cjs/src/query.js:36:24)
        at sql (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/postgres/cjs/src/index.js:111:11)
        at usdEquivalent (/home/runner/work/dimension-adapters/dimension-adapters/fees/first-crypto-bank/index.ts:55:38)
    Failed to get value on ethereum: Error: connect ECONNREFUSED ::1:5432
    ETHEREUM 👇
    Timestamp attempted: 22/12/2023
    Error: connect ECONNREFUSED ::1:5432
        at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
        at cachedError (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/postgres/cjs/src/query.js:171:23)
        at new Query (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/postgres/cjs/src/query.js:36:24)
        at sql (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/postgres/cjs/src/index.js:111:11)
        at usdEquivalent (/home/runner/work/dimension-adapters/dimension-adapters/fees/first-crypto-bank/index.ts:55:38)
    
    
    
    
    Done in 4.57s.

File: fees/first-crypto-bank/index.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          block_number,
+          block_time,
+          "value" / 1e18 as eth_value,
+          encode(transaction_hash, 'hex') AS HASH,
+          encode(to_address, 'hex') AS to_address
+        FROM
+          ethereum.traces
+        WHERE
+          block_number > 17539904
+          and to_address = '\\x67262A61c0A459Fff172c22E60DBC730393BF790'
```
Comment (dtmkeng) [inline fees/first-crypto-bank/index.ts:66]: could you check this address, i check it is a wellet not contract there is not trace, i revice endpoint for this protocol not sure is it  correct `https://api.firstcryptobank.capital/reports/revenue_by_day`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: dexs/raydium/index.ts [hunk 2]
```diff
@@ -16,8 +16,8 @@ const graphs = async (timestamp: number): Promise<FetchResultVolume> => {
   const ammPool: IAmmPoool = (await fetchURL(urlAmmPool)).data;
   const ammPoolStandard: any[] = (await fetchURL(urlAmmPoolStandard)).data.data;;
   const ammPoolStandardVolume: IAmmPooolStandar[] = ammPoolStandard.map((e: any) => e.day);
-  // const dailyVolumeAmmPool = ammPoolStandardVolume.reduce((a: number, b: IAmmPooolStandar) => a + b.volume, 0)
-  const dailyVolumeAmmPool = 0; // office web site show only volume on amm pool
+  const dailyVolumeAmmPool = ammPoolStandardVolume.reduce((a: number, b: IAmmPooolStandar) => a + b.volume, 0)
```
Comment (dtmkeng) [inline dexs/raydium/index.ts]: it disable because raydium web site show only ammpool volume

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: fees/first-crypto-bank/index.ts (hunk 1 above)
Comment (alphadeveth) [reply fees/first-crypto-bank/index.ts:66]: @dtmkeng can forget about endpoint... adapter is good. address is where revenue is collected

**Comment by dtmkeng** [comment]
@alphadeveth  could you resolve conflicts ?

**Comment by alphadeveth** [comment]
> @alphadeveth could you resolve conflicts ?

removed conflict in raydium change, please verify now

**Comment by dtmkeng** [comment]
is there details for list (website, x) ?

**Comment by alphadeveth** [comment]
> is there details for list (website, x) ?

https://firstcryptobank.capital/

**Comment by alphadeveth** [comment]
> is there details for list (website, x) ?

Hello thank you for prompt listing, can we have token link added as well? 

https://etherscan.io/token/0x9360c489056b64d5003bf22f4f31458e31cc8028

**Comment by dtmkeng** [comment]
> > is there details for list (website, x) ?
> 
> Hello thank you for prompt listing, can we have token link added as well?
> 
> https://etherscan.io/token/0x9360c489056b64d5003bf22f4f31458e31cc8028

how meta you need track token (ex amout of token tranter form x to y) ?

**Comment by alphadeveth** [comment]
Hello, we just rebranded token, anyway we can change name from First Crypto Bank to "Bank AI" ? Wallet for tracking revenue is the same.

**Comment by alphadeveth** [comment]
thank you for the quick update, how can we also update logo and branding in "about"?

**Comment by dtmkeng** [comment]
> thank you for the quick update, how can we also update logo and branding in "about"?

could you can drop info  at our discord  have someone working on it

**Pull request merged**